.github/workflows/deploy_staging_portal.yml

```diff
@@ -28,33 +46,23 @@ jobs:
       - name: Build Interface
         id: build
         uses: ./.github/actions/build-interface
+        env:
+          LOKALISE_PROJECT_ID: ${{ vars.LOKALISE_PROJECT_ID }}
+          LOKALISE_API_TOKEN: ${{ secrets.LOKALISE_API_TOKEN }}
         with:
           interfacePath: ${{ env.workingDirectory }}
-          envContentSecurityPolicy: >-
-            connect-src 'self' https://staging.121.global https://westeurope-1.in.applicationinsights.azure.com https://westeurope.livediagnostics.monitor.azure.com https://login.microsoftonline.com;
-            form-action https://staging.121.global;
-            frame-src blob: 'self' https://app.powerbi.com https://login.microsoftonline.com;
-            frame-ancestors 'self' https://flex.twilio.com;
-        env:
-          NG_ENV_NAME: 'staging'
-          NG_LOCALES: ${{ vars.NG_LOCALES }}
-          NG_DEFAULT_LOCALE: ${{ vars.NG_DEFAULT_LOCALE }}
-          NG_URL_121_SERVICE_API: 'https://staging.121.global/api'
-          NG_AI_IKEY: '27eaf00e-92e5-46b7-8d0b-1b413db898e7'
-          NG_AI_ENDPOINT: 'https://westeurope-1.in.applicationinsights.azure.com/'
-          APPLICATIONINSIGHTS_CONNECTION_STRING: 'InstrumentationKey=27eaf00e-92e5-46b7-8d0b-1b413db898e7;IngestionEndpoint=https://westeurope-1.in.applicationinsights.azure.com/;LiveEndpoint=https://westeurope.livediagnostics.monitor.azure.com/'
-          USE_SSO_AZURE_ENTRA: ${{ vars.USE_SSO_AZURE_ENTRA }}
-          AZURE_ENTRA_CLIENT_ID: ${{ vars.AZURE_ENTRA_CLIENT_ID }}
-          AZURE_ENTRA_TENANT_ID: ${{ vars.AZURE_ENTRA_TENANT_ID }}
-          AZURE_ENTRA_URL: ${{ vars.AZURE_ENTRA_URL }}
+          buildType: 'portalicious'
```

Comment:
FYI, this prompted me to create a follow up https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/35758